README.md

```diff
@@ -36,12 +36,26 @@ mkdir -p build/src
 git clone --recurse-submodules https://github.com/ungoogled-software/ungoogled-chromium-debian.git
 # Replace TAG_OR_BRANCH_HERE with the tag or branch you want to build
 git checkout --recurse-submodules TAG_OR_BRANCH_HERE
+
+# NOTE: If you are reading this on GitHub, make sure to read the version corresponding
+# to your checkout of this repo (replace TAG_OR_BRANCH_HERE with the tag or branch you want to build):
+# https://github.com/ungoogled-software/ungoogled-chromium-debian/blob/TAG_OR_BRANCH_HERE/README.md
+#
+# Or, just read the README in your local repo.
+
 cp -r ungoogled-chromium-debian/debian build/src/
 cd build/src
 
 # Final setup steps for debian/ directory
 ./debian/rules setup-debian
 
+# Add packages for LLVM 8
+# The easiest way to do this is to use the APT repo from apt.llvm.org
+# 1. Add this line to your /etc/apt/sources.list: deb http://apt.llvm.org/buster/ llvm-toolchain-buster-8 main
```

Comment:
This should probably be `cosmic` instead of `buster`

Reply:
done

Reply:
see https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/8/commits/ac16b1edaceb9082caed7d3ee6b30806440de35c